README.md

```diff
@@ -126,11 +153,41 @@ The file must be named according to the following rules:
 Good: `columnists.svg`, `back-arrow.svg`
 Bad: `RightArrow.svg`, `linked_in.svg`, `yahoo!.svg`
 
-### IE7 support
-
-IE7 support is handled by an IE7 CSS expression hack, which is bundled into the main icons mixin.
+## Deprecation process
+
+Icon sets can't be versioned so, when removing an icon, keep in mind that it isn't used anywhere. To find out if they're being used, search on [Splunk](http://splunk.internal.ft.com). An example search:
+
+> (host="ftweb61759-law1b-eu-p" OR host="ftweb61758-law1a-eu-p") source="/var/log/httpd/access_log" fticon:section-world| top limit=120 referer
+
+Where you can change _section-world_ for the icon you're looking for.
+
+The following list of icons are deprecated and will be removed in next major release:
```

Comment:
(As discussed) lets move this list to _deprecated.js or something and then use it to exclude deprecated icons from demos. This will mean they're still available to services already using them via the image service but aren't as discoverable by new users.